### DISCUSSION TITLE
Any reason for changing `yobasystems/alpine-mariadb` to `mariadb`

### DISCUSSION THREAD

**Discussion by azzamsa** [firefly-iii/firefly-iii#5515] [Firefly III] [answered]
Hi.

I don't see the "why" (reason) In the commit message nor in file changes.

- https://github.com/firefly-iii/docker/commit/845841a3d57e330e156a9c5d17b552f2455ac043

Is this because [Why don't you provide an Alpine-based image?](https://github.com/MariaDB/mariadb-docker#why-dont-you-provide-an-alpine-based-image)?

**Comment by JC5** [comment, marked as answer]
Yes, I think so. I can't remember. The old, multi-arch image did have a lot of problems.
